R/retro_wrapper.R

```diff
@@ -58,17 +58,16 @@ retro_wrapper <- function(mydir,  model_settings) {
   )
   message("Running retrospectives.")
 
-  r4ss::SS_doRetro(
-    masterdir = retro_dir, 
-  	overwrite = model_settings$overwrite,
-    exefile = model_settings$model, 
+  r4ss::retro(
+    dir = retro_dir, 
+    oldsubdir = model_settings$oldsubdir, 
+    newsubdir = model_settings$newsubdir,  	
+    subdirstart = "retro",
```

Comment:
`subdirstart = "retro"` is the default in r4ss::retro() so you could just delete this line